source/support.tex

```diff
@@ -6164,6 +6164,6 @@
 The header \libheader{stdlib.h}
 assuredly provides the same declarations and definitions within
 the global namespace,
-much as in the C Standard. It may also provide these names within
+much as in \IsoC{}. It may also provide these names within
```

Comment:
Make that undated? We don't refer to a specific clause/annex.

Reply:
Hm, we also don't have an undated entry in the normative references, which is perhaps a problem: If we have both dated and undated ones, we might end up depending  on multiple versions of C. Maybe we can avoid the undated references entirely?

Reply:
This is a note / example.  Everywhere else we use undated refs, we are also non-normative or explanatory, I think. So it seems we're good.

Reply:
OK, let's try. We do actually use an undated C reference normatively in one place (with the floating point types) :-S